common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java

```diff
@@ -105,6 +111,24 @@ public void init(OperationMeta operationMeta) {
     setAbsolutePath(concatPath(swagger.getBasePath(), operationMeta.getOperationPath()));
   }
 
+  private void checkDownloadFile(Operation operation) {
```

Comment:
1.change name to checkDownloadFileFlag
2.OperationMeta already collect ResponseMetas, just invoke org.apache.servicecomb.swagger.invocation.response.ResponsesMeta#findResponseMeta, and determined by org.apache.servicecomb.swagger.invocation.response.ResponseMeta#javaType